### PR TITLE
Street mode translation in zh-tw

### PR DESCRIPTION
**Pull request by Lafudoci**
Translated `street mode` and revised the `show secrets`.
@0140454, could you kindly help to review this? BTW, please note that `street mode` was translated by private mode instead of literal translating.

### CODE REVIEW

**Comment by m2049r** [comment]
@Lafudoci after consulting with my design team, we feel "public mode" is better, as "private mode" is the default mode of monero. sorry about the confusion.

**Comment by Lafudoci** [comment]
@m2049r, I agree with that private mode might confuse people, but public mode could have the same problem IMO. It literally indicates something of Monero wallet will be exposed. How about "low profile mode"? I believe it's more precise.

**Comment by 0140454** [comment]
IMHO, i think "low profile mode" is a good choice.

File: app/src/main/res/values-zh-rTW/help.xml [hunk 1]
```diff
@@ -171,11 +171,10 @@
 
     <string name="help_wallet"><![CDATA[
         <h1>錢包介面</h1>
-        <h2>Street Mode</h2>
-        <p>Street mode can be enabled/disabled in the menu or Gunther\'s head icon. In this mode, your
-        balance is not shown on any screen so you can safely use your wallet on the street, a pub or
-        other public place. Previous transactions are also hidden. New transactions will be shown, so
-        you can see that you have sent/received sweet Moneroj!</p>
+        <h2>隱私模式</h2>
+        <p>隱私模式可以藉由點選 Gunther 的頭像或在選單中啟用/關閉。在這模式中你
```
Comment (0140454) [inline app/src/main/res/values-zh-rTW/help.xml]: I think there should be a newline after the period, or there will be a space between 你 and 的. ([ref](https://github.com/Lafudoci/xmrwallet/pull/1))

```
隱私模式可以藉由點選 Gunther 的頭像或在選單中啟用/關閉。
在這模式中你的帳戶餘額不會顯示在任何的畫面中，因此你可以安全的在街上、酒吧等公共場所使用錢包。
```

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 2]
```diff
@@ -353,7 +353,7 @@
 
     <string name="status_wallet_connect_wrongversion">節點版本不相容 - 請盡速更新！</string>
 
-    <string name="menu_info">詳細資訊</string><!--Changed to: Show Secrets!-->
-    <string name="menu_streetmode">Street Mode</string>
-    <string name="info_streetmode_enabled">Street Mode enabled, tap for more info.</string>
+    <string name="menu_info">顯示私鑰資訊</string><!--Changed to: Show Secrets!-->
```
Comment (0140454) [inline app/src/main/res/values-zh-rTW/strings.xml]: Should we remove the comment after revising?

File: app/src/main/res/values-zh-rTW/help.xml (hunk 1 above)
Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/help.xml]: OK, Thanks, I'll revise it once the term "street mode" is decided.

File: app/src/main/res/values-zh-rTW/strings.xml (hunk 2 above)
Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/strings.xml]: I'm not sure either, maybe ping @m2049r

Comment (erciccione) [reply app/src/main/res/values-zh-rTW/strings.xml]: I would remove it. I think he put it there to guide translators, but it lose its meaning once that string is correctly translated. Let's wait for him anyway

Comment (m2049r) [reply app/src/main/res/values-zh-rTW/strings.xml]: yes, what @erciccione said :)

**Comment by anhdres** [comment]
Please note it's not called _public mode_ but _street mode_.
I agree it would be confusing the public/private naming, that's why we went for something that easily described hte "use case" instead of the "characteristics" of the mode. We assume the street is public and the _street mode_ is something you should use if in a public setting.
If there's a nuanced alternative for each language, I think it should keep this difference in mind.

**Comment by Lafudoci** [comment]
@anhdres, I agree with you. However, the "street mode" (街道模式) or something like that in traditional Chinese are just weird and not intuitive (at least to me). And I believe we already have some consensus in this thread that we use "low profile mode" (低調模式) instead could be a acceptable choice. But I can still make a quick survey for opion from more Taiwanese people.

**Comment by anhdres** [comment]
I think _low profile mode_ is perfect. Love it.

Thanks for your contributions btw!

**Comment by Lafudoci** [comment]
@m2049r, It seems we have good consensus. The [quick survey](https://www.facebook.com/groups/MoneroTaiwan/permalink/262891251090285/) in Monero TW FB community also favored low profile mode. So I have made commit for it and included some minor fix reported by @0140454 .

File: app/src/main/res/values-zh-rTW/help.xml [hunk 3]
```diff
@@ -171,11 +171,10 @@
 
     <string name="help_wallet"><![CDATA[
         <h1>錢包介面</h1>
-        <h2>Street Mode</h2>
-        <p>Street mode can be enabled/disabled in the menu or Gunther\'s head icon. In this mode, your
-        balance is not shown on any screen so you can safely use your wallet on the street, a pub or
-        other public place. Previous transactions are also hidden. New transactions will be shown, so
-        you can see that you have sent/received sweet Moneroj!</p>
+        <h2>隱私模式</h2>
```
Comment (0140454) [inline app/src/main/res/values-zh-rTW/help.xml]: 「隱私模式」 should change to 「低調模式」.

Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/help.xml]: oops, my bad. I'll fix soon.

Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/help.xml]: Fixed

**Review by 0140454 — APPROVED** [review]

**Review by m2049r — APPROVED** [review]

**Pull request merged**